src/execution/compiler/operator/seq_scan_translator.cpp

```diff
@@ -36,6 +36,11 @@ SeqScanTranslator::SeqScanTranslator(const planner::SeqScanPlanNode &plan, Compi
     local_filter_manager_ = pipeline->DeclarePipelineStateEntry("filterManager", fm_type);
   }
 
+  tvi_base_ =
```

Comment:
Why do we need to make these changes to the seq_scan_translator? As far as I can tell we don't call abort from anywhere within the translator.

Reply:
A seq scan translator can "nest" (`ctx->Push()`, (line 227)) other translators inside it.